### PR TITLE
Changed UI to gov style

### PR DESCRIPTION
**Pull request by liviahull**
## Pull request overview

Give a general description of why a change is being made, include issue number(s) being fixed if relevant

## Pull request checklist

Please check if your PR fulfils the following:
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)

## What is the current behaviour?

The UI relies on shinydashboard package, has the navigation between pages through the tabs on the side bar, above the selection boxes for different categories (Region, Industry Sector and so on). Layouts are based on boxes where tables are charts are placed. 

image

## What is the new behaviour?

The UI has changed to line up with the LEO dashboard  to be published May 2022 and govstyle (css file is in the www/ folder, and the footer is from the shinyGovstyle package).

homepage

qualpathways

support

## Anything else

In addition to this, the collapsible tree on page 2 has got a legend above the chart. I added a page for Homepage, one for Accessibility and one for _Support and Feedback, all to be revised as they are placeholders for now.

### CODE REVIEW

**Comment by Jkhamis** [comment]
Thanks Livia - this looks great. Couple of minor comments:

- In the Industry Sector Overview. Can you add some text below the KPI boxes to describe the tabs? The text should be: "Select the sub-sector and level tab for a breakdown of employee numbers and median earnings by highest level of education and detailed industry sector. Select the subject and qualification tab for more information about the highest qualifications employees hold.
- Again in the industry sector overview tab - there is a bit of space between the sector and level charts and the bottom of the pages. If this is still there with the added text then could you increase the height of the charts a little to fill it?  
- In the qualification pathways section - can you add a "how to read this chart" section under the title? I think we need to add a  short explanation there.

All my other comments are about completing the text - so that can wait. 

Thanks!
Jay

**Review by Jkhamis — APPROVED** [review]

File: R/accessibility_statement.R [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+           are known issues that we will continue to monitor and improve:",
+    tags$div(tags$ul(
+      tags$li("Keyboard navigation through the interactive charts is currently limited"),
+      tags$li("Alternative text in interactive charts is limited to titles and could be more descriptive (although this data is available in csv format)")
+    )), # TODO: UPDATE THESE BULLET POINTS
+    
+    h3("Feedback and contact information"),
+    "If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille or
+      if you have any feedback on how we could further improve the accessibility of this application, please contact us at ",
+    a(href = "mailto:he.leo@education.gov.uk", "he.leo@education.gov.uk", .noWS = c("after")),
```
Comment (cjrace) [inline R/accessibility_statement.R:36]: Should the email address be updated?

File: R/support_links.R [hunk 2]
```diff
@@ -0,0 +1,32 @@
+support_links <- function() {
+  div(
+    h2("Give us feedback"),
+    "This dashboard is a new service that we are developing. If you have any feedback or suggestions for improvements, please submit them using our ",
+    a(
+      href = "https://forms.office.com/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-c6JT6ONG3lJtlg-5hU4A6xURUpQME1OUVZIMEFMUUdNMEVONkhEN0g1VSQlQCN0PWcu",
```
Comment (cjrace) [inline R/support_links.R]: This needs updating to be a form for this specific dashboard, can create a form using microsoft forms with similar questions to the SCAP form - https://forms.office.com/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-fim2cxpeIFFus-loZuDRkhUN0ZPMlhCSDgzQjlVTjNVTFpXRTdWMlo0MC4u

File: R/support_links.R [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    ".",
+    br(),
+    h2("Find more information on the data"),
+    "The data used to produce the dashboard, along with methodological information can be found on ",
+    a(href = "https://explore-education-statistics.service.gov.uk/", "Explore Education Statistics", .noWS = c("after")),
+    ".",
+    br(),
+    h2("Contact us"),
+    "If you have questions about the dashboard or data within it, please contact us at ",
+    a(href = "mailto:he.leo@education.gov.uk", "he.leo@education.gov.uk", .noWS = c("after")),
```
Comment (cjrace) [inline R/support_links.R:18]: Does this need updating?

File: R/support_links.R [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    ".",
+    br(),
+    h2("See the source code"),
+    "The source code for this dashboard is available in our ",
+    a(href = "https://github.com/dfe-analytical-services/leo-graduate-industry-dashboard", "GitHub repository", .noWS = c("after")),
```
Comment (cjrace) [inline R/support_links.R]: Needs updating to this repository

**Review by cjrace — CHANGES_REQUESTED** [review]
Looks good - nice work on getting it all working with the new styling!

Few thoughts from looking around:

- I'd suggest removing the custom formatting from the tabs so that they're consistent with other dashboards, currently it feels like they stick out a bit too much.
- The reset button on the subject and qualification tab seems a bit lost / out of place. It should probably sit within the blue sidebar, though even then it seems like users might miss it if it's at the bottom of the page?
- The font in the drop downs seems to be different to standard and looks like it's aligned strangely?
This branch:
image
LEO dashboard (as an example of what I was expecting)
image
- Link from the homepage to qualification pathways doesn't seem to be working? (the other link above for industry sector overview does work)

File: R/accessibility_statement.R (hunk 1 above)
Comment (liviahull) [reply R/accessibility_statement.R:36]: Yes but we don't have one yet - it will be done at later stage. All above addressed now and solved.

**Comment by rmbielby** [comment]
Don't think I've got much extra beyond Cam's comments. Like I said in the previous PR, I think the legend circles could be bigger. We could also add the black ring around each. So something like:
```
  svg_html_legend <- paste('<svg width="450" height="20">',
                           '<circle cx="10" cy="10" r="8" style="fill: rgb(29, 112, 184);"></circle>',
                           '<circle cx="90" cy="10" r="8" style="fill: rgb(0, 48, 120);"></circle>',
                           '<circle cx="170" cy="10" r="8" style="fill: rgb(145, 43, 136);"></circle>',
                           '<circle cx="260" cy="10" r="8" style="fill: #4c2c92";></circle>',
                           '<circle cx="340" cy="10" r="8" style="fill: #28a197";></circle>',
                           '<circle cx="10" cy="10" r="8" style="fill: none; stroke: black; stroke-width: 2;"></circle>',
                           '<circle cx="90" cy="10" r="8" style="fill: none; stroke: black; stroke-width: 2;"></circle>',
                           '<circle cx="170" cy="10" r="8" style="fill: none; stroke: black; stroke-width: 2;"></circle>',
                           '<circle cx="260" cy="10" r="8" style="fill: none; stroke: black; stroke-width: 2;"></circle>',
                           '<circle cx="340" cy="10" r="8" style="fill: none; stroke: black; stroke-width: 2;"></circle>',
                           '<text x="25" y="12" alignment-baseline="middle" style="font-size: 15px;">Level 2</text>',
                            '<text x="105" y="12" alignment-baseline="middle" style="font-size: 15px;">Level 3</text>',
                            '<text x="185" y="12" alignment-baseline="middle" style="font-size: 15px;">Level 4/5</text>',
                            '<text x="275" y="12" alignment-baseline="middle" style="font-size: 15px;">Level 6</text>',
                            '<text x="355" y="12" alignment-baseline="middle" style="font-size: 15px;">Level 7+</text>')


```

File: R/support_links.R (hunk 3 above)
Comment (liviahull) [reply R/support_links.R:18]: Yes it does but we don't have one yet.

File: R/support_links.R (hunk 4 above)
Comment (liviahull) [reply R/support_links.R]: This has been updated.

**Comment by liviahull** [comment]
@cjrace and @rmbielby I have addressed all these. In addition I added a column for level to the table. Data will show a level for each qualification regardless of "All levels" selection in the filter.

**Review by cjrace — APPROVED** [review]
Thanks Livia - happy to approve now

**Pull request merged**